tests/lib/rules/no-tabs.js

```diff
@@ -23,11 +23,46 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n}",
         "function test(){\n" +
         "  //   sdfdsf \n" +
-        "}",
+        "}"
     ],
     invalid: [
+
+        // Tabs in string/template/regex literals should be reported but not auto-fixed
```

Comment:
Well, you would want to insert the literal string `\t` into the source code, so you would do something like:

```js
context.replaceTextRange(range, '\\t');
```

(I think this is what you were saying.)